desktop-wm/cwm/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=cwm
+PKGSEC=x11
+PKGDEP="x11-lib"
+BUILDDEP="pkg-config bison"
+PKGDES="A lightweight and efficient window manager for X11"
```

Comment:
"Efficient" is an empty word, I would just cut it. Perhaps consider rewriting the description to showcase the key features of this window manager when compared to other "lightweight" options.